### PR TITLE
Deployment Potion Points

### PR DESCRIPTION
**Pull request by 0xm3rlin**

### CODE REVIEW

File: foundry.toml [hunk 1]
```diff
@@ -50,6 +50,9 @@ dictionary_weight = 40
 include_storage = true
 include_push_bytes = true
 
+[etherscan]
+arbitrum = {key = "${ARBISCAN_TOKEN}"}
```
Comment (0xCalibur) [inline foundry.toml]: why was this necessary?

Comment (0xm3rlin) [reply foundry.toml]: Maybe it is not, I was trying to fix some issues during deployment
